balance_check/providers/gamestop.py

```diff
@@ -0,0 +1,111 @@
+import sys
+import requests
+from bs4 import BeautifulSoup
+from balance_check import logger, captcha_solver, config
+from balance_check.providers import BalanceCheckProvider
+from balance_check.validators.gift_card import Merchant, GiftCardSchema
+
+
+class GameStop(BalanceCheckProvider):
+    def __init__(self):
+        super().__init__()
+
+        self.website_url = "https://www.gamestop.com/profiles/valuelookup.aspx"
+        self.schema = GiftCardSchema([Merchant.GameStop])
```

Comment:
Signature change, list -> single merchant.
```suggestion
        self.schema = GiftCardSchema(Merchant.GameStop)
```